supervisor/utils/log_format.py

```diff
@@ -16,6 +17,6 @@ def format_message(message: str) -> str:
             return f"Port '{match.group(1)}' is already in use by something else on the host."
     except TypeError as err:
         sentry_sdk.capture_exception(err)
-        raise FormatError("Type of message is not string") from None
+        _LOGGER.error("Type of message is not string - %s", err)
```

Comment:
let's print the error and capture after that the exception, so we get all in one sentry